<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Oscilloscope</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/thin_oscilloscope-512.png</normaloff>img/thin_oscilloscope-512.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: white;
}

QWidget#container, #container_2{
	background-color: black
}

QWidget#container QLabel, #container QCheckBox, #container_2 QLabel{
	color: white;
	font-size: 14px;
	font-family: monospace
}
QGroupBox#timeGroupBox{
	border: 2px solid #d1c600;
	border-radius: 5px;
	background-color: #f5f5f5
}
QGroupBox#freqGroupBox{
	border: 2px solid #3dc6e4;
	border-radius: 5px;
	background-color: #f5f5f5
}
QLabel{
	border: none
}

QPushButton{
	border: none;
	padding: 5px
}

QPushButton:checked {
    background-color: rgba(67, 67, 67, 40)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>851</width>
      <height>651</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="screens">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QWidget" name="container_2" native="true">
       <widget class="Plot" name="timeGraph" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>660</width>
          <height>290</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>CrossCursor</cursorShape>
        </property>
        <widget class="QLabel" name="label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>570</x>
           <y>10</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>0.000 s</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="container" native="true">
       <widget class="Plot" name="freqGraph" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>660</width>
          <height>290</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>CrossCursor</cursorShape>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>570</x>
           <y>10</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>000.0 Hz</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </widget>
       <widget class="QCheckBox" name="log">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>290</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Log</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="timeGroupBox">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>40</y>
      <width>360</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Input</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>161</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>INPUT RATE:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="rate"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>INPUT CHUNK SIZE:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="chunk"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>DEPTH:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="depth"/>
      </item>
     </layout>
    </widget>
    <widget class="QDial" name="in_gain">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>50</width>
       <height>64</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Gain</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="freqGroupBox">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>350</y>
      <width>360</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <widget class="QPushButton" name="play_sound">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>230</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/play1.png</normaloff>img/play1.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>30</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="sin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>img/sine.png</normaloff>img/sine.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="triangular">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>img/triangular.png</normaloff>img/triangular.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="square">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>img/squared.png</normaloff>img/squared.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sawtooth">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>img/sawtooth.png</normaloff>img/sawtooth.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>170</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Gain</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>160</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Frecuencia:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="freq">
        <property name="text">
         <string>440</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Hz</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QDial" name="gen_dial">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>50</width>
       <height>64</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Plot</class>
   <extends>QWidget</extends>
   <header>plot</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
